relayer/chains/cosmos/module/app_module.go

```diff
@@ -0,0 +1,42 @@
+package module
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/spf13/cobra"
+
+	tmlightclient "github.com/cosmos/ibc-go/v7/modules/light-clients/07-tendermint"
+)
+
+// AppModuleBasic defines the basic application module used by the module.
+type AppModuleBasic struct{}
+
+// Name returns the module's name.
+func (AppModuleBasic) Name() string {
+	return "cosmos_chain_provider"
+}
+
+// RegisterLegacyAminoCodec does nothing. IBC does not support amino.
+func (AppModuleBasic) RegisterLegacyAminoCodec(*codec.LegacyAmino) {}
+
+// RegisterInterfaces registers module concrete types into protobuf Any.
+func (AppModuleBasic) RegisterInterfaces(registry codectypes.InterfaceRegistry) {
+	tmlightclient.RegisterInterfaces(registry)
```

Comment:
This entire file is necessary because of changes in v7 of ibc-go right?

Reply:
Yes, only reason is to register tm light client codecs as they are no longer registered by default in v7